tool_collections/kraken/kraken_report/.shed.yml

```diff
@@ -0,0 +1,14 @@
+owner: devteam
+name: kraken_report
+description: Contains galaxy tool wrapper for kraken-report.
+categories:
+  - Metagenomics
+long_description: 'Kraken is a system for assigning taxonomic labels to short DNA
```

Comment:
Does this work in yaml? I thought we need to use `|` to indicate a multi-line text.